mod_tariffs.asciidoc

```diff
@@ -774,111 +907,111 @@ the time based cost of an expired reservation duration will be the price in the
-                                                         This restriction can make a TariffElement become active when the charging current is below this value,
-                                                         but the TariffElement is no longer active when charging comes above this current.
-|min_power |<<types.asciidoc#types_number_type,number>> |? |Minimum power in kW, for example 5. When the Charging Session is consuming more then this amount of power,
-                                                            this TariffElement is/becomes active. Is the charging power is lower, this TariffElement is not valid.
-                                                            This is NOT the minimum power over the entire Charging Session.
-                                                            This restriction can make a TariffElement become active when the charging power is above this value,
-                                                            but the TariffElement is no longer active when charging drops below this power.
-|max_power |<<types.asciidoc#types_number_type,number>> |? |Maximum power in kW, for example 20. When the Charging Session is consuming less then this amount of power,
-                                                         this TariffElement becomes/is active. Is the charging power is higher, this TariffElement is not valid.
-                                                         This is NOT the maximum power over the entire Charging Session.
-                                                         This restriction can make a TariffElement become active when the charging power is below this value,
-                                                         but the TariffElement is no longer active when charging comes above this power.
-|min_duration |int |? |Minimum duration in seconds the Charging Session is ongoing. When the duration of the ongoing Charging Session is longer then this value,
-                      this TariffElement is active, before that moment this TariffElement is not yet active. Duration in seconds.
-|max_duration |int |? |Maximum duration in seconds the Charging Session is ongoing. When the duration of the ongoing Charging Session is shorted then this value,
-                      this TariffElement is active, after that moment this TariffElement is no longer active. Duration in seconds.
+|start_time |<<types.asciidoc#types_string_type,string>>(5) |? |Start time of day, for example 13:30, valid from this time of the day. Must be in 24h format with leading zeros. Hour/Minute separator: ":" Regex: `[0-2][0-9]:[0-5][0-9]`
+|end_time |<<types.asciidoc#types_string_type,string>>(5) |? |End time of day, for example 19:45, valid until this time of the day. Same syntax as `start_time`.
+|start_date |<<types.asciidoc#types_string_type,string>>(10) |? |Start date, for example: 2015-12-24, valid from this day. Regex: `([12][0-9]{3})-(0[1-9]|1[0-2])-(0[1-9]|[12][0-9]|3[01])`
```

Comment:
Does this regex render correctly in asciidoc? we might need some escape characters. The GitHub preview does not work with this. Have not check pdf generation.

Reply:
I haven't checked PDF generation either, but there are already other regex patterns in `mod_locations.asciidoc` which use the same characters except for `|`. As far as I know, the pipe is only used for tables and they require it to be placed on the first character of a line, which is not the case here. Can we simply try it?

Reply:
Using regex is not a problem, using `|` in a table is the issue.
Pulled your branch and tried to generate the PDF, sorry, the table is broken.

Reply:
Tried solving it, the `|` needs to be escaped. Change it to: '\|'
change the regex to: `([12][0-9]{3})-(0[1-9]\|1[0-2])-(0[1-9]\|[12][0-9]\|3[01])` that works